zulip/zulip/examples/get-stream-topics

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+
+usage = """get-stream-topics --stream-id=<stream-id> [options]
+
+Get all the topics for a specific stream.
+"""
+
+import zulip
+
+parser = zulip.add_default_arguments(argparse.ArgumentParser(usage=usage))
+parser.add_argument('--stream-id')
```

Comment:
This should have been tagged as `required=True`.  

I also suspect that from a user perspective, we should probably have this example do two calls: First one to get the stream from name->ID, then another to do the actual thing (since on the command line, you probably have a name).